### ISSUE TITLE
Program won't work for slanting lines

### ISSUE THREAD

**Issue by juleshwar** [manovenkatesh/programming-questions#1]
This program only addresses LOS issues while happen perpendicularly or horizontally!
e.g
.....................A..........
..................../............
..................C-----D
................/...............
.............B..................

**Comment by manovenkatesh** [comment]
yeah...!! i know bro. I wrote seperate code for that, but in given online compiler  it is not containing any such 
test cases thats why i didnt include that cases.
